cli/command_user_add_set.go

```diff
@@ -35,6 +37,7 @@ func (c *commandServerUserAddSet) setup(svc appServices, parent commandParent, i
 	}
 
 	cmd.Flag("ask-password", "Ask for user password").BoolVar(&c.userAskPassword)
+	cmd.Flag("key-derivation-algorithm", "Key Derivation Algorithm").Default(crypto.DefaultKeyDerivationAlgorithm).StringVar(&c.keyDerivationAlgorithm)
```

Comment:
Same suggestion here:
```suggestion
	cmd.Flag("key-derivation-algorithm", "Key Derivation Algorithm").Default(crypto.DefaultKeyDerivationAlgorithm).EnumVar(&c.keyDerivationAlgorithm, crypto.AllowedKeyDerivationAlgorithms()...)
```